### ISSUE TITLE
Add a save button

### ISSUE THREAD

**Issue by cleanyoursockets** [enzo1982/freac#426]
**Describe the solution you'd like**
Add a save button.

**Additional context**
Adding a save button, would make it possible to edit already, ripped CD's.

**Comment by enzo1982** [comment]
Yes, I have plans to add a save button to the _Tracks_ and _Albums_ tabs of the tag editor at some point in order to make this possible.

**Comment by cleanyoursockets** [comment]
Great news, doing all the ripping and editing in your app, would be very helpful, thanks.

**Comment by Lee-Carre** [comment]
> Add a save button

While I don't disagree, I'm mindful of Fre:AC becoming bloated if it's expected to become a generalised tag editor.

[Picard](//picard.musicbrainz.org/) might be better suited to mass-editing an existing library.
